app/scripts/metamask-controller.js

```diff
@@ -733,113 +897,37 @@ module.exports = class MetamaskController extends EventEmitter {
 // END (VAULT / KEYRING RELATED METHODS)
 //=============================================================================
 
-//
-
-//=============================================================================
-// MESSAGES
-//=============================================================================
-
+  /*
```

Comment:
Block comments technically need to begin with two asterisks to highlight properly on GH (and potentially parsed properly by some JSDoc parsers.) This comment applies throughout the PR.